### PR TITLE
NMS-16982: Updating the SCV Page in the Meridian 2023 Docs 

### PR DESCRIPTION
**Pull request by garcia-sr**
### All Contributors

* [ x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://opennms.atlassian.net/) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the Jira link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the Jira issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* Jira (Issue Tracker): https://opennms.atlassian.net/browse/NMS-16982

### CODE REVIEW

**Review by dino2gnt — APPROVED** [review]
Content approved, but I can't verify if all the referenced configs have access to the SCV scope in M2023.  Maybe Ronny knows?

**Review by indigo423 — APPROVED** [review]
Thank you for the improvements!🍻

File: docs/modules/operation/pages/deep-dive/meta-data.adoc [hunk 1]
```diff
@@ -108,6 +108,9 @@ The following keys are available under this context:
 | interface:if-description
 | The SNMP interface description
 
+| interface:if-name
```
Comment (cgorantla) [inline docs/modules/operation/pages/deep-dive/meta-data.adoc:111]: `if name` was only added by this PR #7515  which was added in foundation-2024

File: docs/modules/operation/pages/deep-dive/meta-data.adoc [hunk 2]
```diff
@@ -254,3 +266,21 @@ The following {page-component-title} components allow the use of the Metadata DS
 * <<reference:configuration/ttl-rpc.adoc#metadata-ttls, Using metadata for TTLs>>.
 * <<deep-dive/thresholds/thresholding.adoc#ga-threshold-metadata, Expression-Based Thresholds>>.
 * <<deep-dive/notifications/introduction.adoc#ga-notifications, Notifications>>.
+* <<deep-dive/alarms/alarm-handling.adoc#ga-advanced-alarm-handling, Alarmd Drools rules>>.
+
+Furthermore, some {page-component-title} configuration files allow the use of Metadata DSL to access the global scope `scv` in order to reference credentials in the Secure Credentials Vault:
+
```
Comment (cgorantla) [inline docs/modules/operation/pages/deep-dive/meta-data.adoc:272]: Are you sure all below configs parse metadata. Poller configs, collector configs I can see that they are supported. Snmp configs I have done it myself. Other configs we have to validate.

Comment (garcia-sr) [reply docs/modules/operation/pages/deep-dive/meta-data.adoc:272]: Yeah, you bring up a good point Chandra. I rushed this thinking that all the SCV meta data interpolation stuff would be in this release and it's not. Dino sent me some of the NMS issues where those features were put into place in H33. Which wouldn't be in Meridian 2023. 

I'm going to close this issue and re-open it for Foundation 2024 instead.

**Comment by garcia-sr** [comment]
Closing and re-opening for Foundation 2024 instead. Foundation 2023 doesn't have all of the SCV features that 2024 has, so this documentation change will have false information.

**Pull request closed without merging**